docs/guides/audio-tracks.md

```diff
@@ -1,69 +1,80 @@
 # Audio Tracks
+Audio tracks are a feature of HTML5 video for providing alternate audio track selections to the user, so that a track other than the main track can be played. Video.js offers a cross-browser implementation of audio tracks.
 
-Audio Tracks are a function of HTML5 video for providing alternative audio track selections to the user, so that a track other than the main track can be played. Video.js makes audio tracks work across all browsers. There are currently five types of tracks:
+## Caveats
+- It is not possible to add audio tracks through HTML. They must be added [programmatically](#api).
+- Literal switching of audio tracks for playback is not handled by Video.js and must be handled by something else. Video.js only stores the track representation.
```

Comment:
maybe it would be a good idea to specify some examples here such as safari supporting them natively and HLS using them to support multiple audio tracks